package.json

```
@@ -25,14 +25,16 @@
     "@subql/cli": "latest",
     "@subql/testing": "latest",
     "@subql/types": "latest",
-    "typescript": "^5.2.2",
-    "moonbeam-types-bundle": "^2.0.8"
+    "moonbeam-types-bundle": "^2.0.8",
+    "typescript": "^5.2.2"
   },
   "resolutions": {
     "ipfs-unixfs": "6.0.6"
   },
   "dependencies": {
     "@polkadot/api-augment": "^10.12.1",
+    "@polkadot/util": "^12.6.2",
+    "@polkadot/util-crypto": "^12.6.2",
     "@subql/types-core": "^0.5.0",
```

```suggestion
    "@subql/types-core": "^0.7.0",
```

that version lead to error:
 ›   Error: [tsl] ERROR in /Users/stepanlavrentev/subquery-proxy/src/types/models/Account.ts(2,58)
 ›         TS2305: Module '"@subql/types-core"' has no exported member 'GetOptions'.
 ELIFECYCLE  Command failed with exit code 2.

---

I've had this issue in the past, but I can't remember what I did to get rid of it. The indexer was running fine for me with 0.5, but there's no problem in bumping it if that solves your issue, I just did it.